### ISSUE TITLE
InvalidInputValue Error Using NLDI navigate_byid()

### ISSUE THREAD

**Issue by kdmonroe** [hyriver/pynhd#61]
**What happened**: 
Following an example notebook - [dam_impact.ipynb]( https://github.com/hyriver/HyRiver-examples/tree/main/notebooks/dam_impact.ipynb) fails at the cell (17) using `navigate_byid()`. All previous cells executed successfully and in sequence. Returns the following error - 

```python 
---------------------------------------------------------------------------
InvalidInputValue                         Traceback (most recent call last)
c:\Users\keonm\Documents\GitHub\HyRiver-examples\Geospatial Hydrologic Data Using Web Services.ipynb Cell 20' in <cell line: 3>()
      [3](vscode-notebook-cell:/c%3A/Users/keonm/Documents/GitHub/HyRiver-examples/Geospatial%20Hydrologic%20Data%20Using%20Web%20Services.ipynb#ch0000019?line=2) for agency, fid in sites[["agency_cd", "site_no"]].itertuples(index=False, name=None):
      [4](vscode-notebook-cell:/c%3A/Users/keonm/Documents/GitHub/HyRiver-examples/Geospatial%20Hydrologic%20Data%20Using%20Web%20Services.ipynb#ch0000019?line=3)     try:
----> [5](vscode-notebook-cell:/c%3A/Users/keonm/Documents/GitHub/HyRiver-examples/Geospatial%20Hydrologic%20Data%20Using%20Web%20Services.ipynb#ch0000019?line=4)         flw_up[fid] = nldi.navigate_byid(
      [6](vscode-notebook-cell:/c%3A/Users/keonm/Documents/GitHub/HyRiver-examples/Geospatial%20Hydrologic%20Data%20Using%20Web%20Services.ipynb#ch0000019?line=5)             fsource="nwissite",
      [7](vscode-notebook-cell:/c%3A/Users/keonm/Documents/GitHub/HyRiver-examples/Geospatial%20Hydrologic%20Data%20Using%20Web%20Services.ipynb#ch0000019?line=6)             fid=f"{agency}-{fid}",
      [8](vscode-notebook-cell:/c%3A/Users/keonm/Documents/GitHub/HyRiver-examples/Geospatial%20Hydrologic%20Data%20Using%20Web%20Services.ipynb#ch0000019?line=7)             navigation="upstreamTributaries",
      [9](vscode-notebook-cell:/c%3A/Users/keonm/Documents/GitHub/HyRiver-examples/Geospatial%20Hydrologic%20Data%20Using%20Web%20Services.ipynb#ch0000019?line=8)             source="flowlines",
     [10](vscode-notebook-cell:/c%3A/Users/keonm/Documents/GitHub/HyRiver-examples/Geospatial%20Hydrologic%20Data%20Using%20Web%20Services.ipynb#ch0000019?line=9)             distance=10)
     [11](vscode-notebook-cell:/c%3A/Users/keonm/Documents/GitHub/HyRiver-examples/Geospatial%20Hydrologic%20Data%20Using%20Web%20Services.ipynb#ch0000019?line=10)     except ZeroMatched:
     [12](vscode-notebook-cell:/c%3A/Users/keonm/Documents/GitHub/HyRiver-examples/Geospatial%20Hydrologic%20Data%20Using%20Web%20Services.ipynb#ch0000019?line=11)         noflw.append(fid)

File ~\miniconda3\envs\pygeo-hyriver\lib\site-packages\pynhd\pynhd.py:945, in NLDI.navigate_byid(self, fsource, fid, navigation, source, distance, trim_start)
    [942](file:///~/miniconda3/envs/pygeo-hyriver/lib/site-packages/pynhd/pynhd.py?line=941)     raise ZeroMatched
    [944](file:///~/miniconda3/envs/pygeo-hyriver/lib/site-packages/pynhd/pynhd.py?line=943) if navigation not in valid_navigations.keys():
--> [945](file:///~/miniconda3/envs/pygeo-hyriver/lib/site-packages/pynhd/pynhd.py?line=944)     raise InvalidInputValue("navigation", list(valid_navigations.keys()))
    [947](file:///~/miniconda3/envs/pygeo-hyriver/lib/site-packages/pynhd/pynhd.py?line=946) url = valid_navigations[navigation]
    [949](file:///~/miniconda3/envs/pygeo-hyriver/lib/site-packages/pynhd/pynhd.py?line=948) r_json = self._get_url(url)

InvalidInputValue: Given navigation is invalid. Valid options are:
description
type
```

**What you expected to happen**: Cell executes + calls NLDI web service using function

**Environment**: Created conda environment using repo's .yml. Installed Jupyter Lab & running in VSCode.

**Comment by cheginit** [comment]
Thanks for reporting the issue. Can you please show the output of:

```python
import pynhd

print(pynhd.__version__)
```

I just ran the notebook with version 0.13.3 and it worked without any issue:
<img width="1135" alt="image" src="https://user-images.githubusercontent.com/13016644/183256843-d0eb90dc-9cf4-4f32-921a-730b28686176.png">

**Comment by kdmonroe** [comment]
![image](https://user-images.githubusercontent.com/46966419/183265731-54158fba-9d6d-4f82-9890-08554c179fad.png)

**Comment by kdmonroe** [comment]
I just tried updating the `pynhd` package via conda but still get the same version as 0.13.2 - and the error persists.

**Comment by cheginit** [comment]
I just remembered the issue that I addressed in 0.13.2. The NLDI web service, which this function uses, has recently been updated and these changes broke this function. So I fixed it in 0.13.3. I am guessing that you are still using the cached response from your previous runs. Please remove the HyRiver's cache directory and try again. By default HyRiver creates a folder called `cache` in your current directory.

**Comment by kdmonroe** [comment]
Success! Rebuilt using 0.13.3 + deleted `cache` dir. Thank you!
